backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/export/svg/EdgeExportService.java

```diff
@@ -328,7 +328,7 @@ private StringBuilder exportBasicDiamondPath(int strokeWidth) {
         basicDiamondPathExport.append(" L "); //$NON-NLS-1$
         basicDiamondPathExport.append(10 + strokeWidth * 2);
         basicDiamondPathExport.append(" 0 L "); //$NON-NLS-1$
-        return basicDiamondPathExport.append(5 + strokeWidth);
+        return basicDiamondPathExport.append(5 + strokeWidth).append(" ").append(3.5 + strokeWidth); //$NON-NLS-1$
```

Comment:
Where does that `3.5` comes from?

Reply:
Without accounting for stroke width the diamond path looks like

`m 0 0 L 5 -3.5 L 10 0 L 5 3.5`

The L 5 3.5 at the end is to reach the bottom corner of the diamond from L 10 0 which is the right corner of the diamond, it's opposed to the top corner reached with the "L 5 -3.5" instruction.

3.5 was missing from the path causing an error. as "L 5" is not a valid path instruction.

Reply:
The 'Z' at the end of the path was missing